### DISCUSSION TITLE
Is the app alive?

### DISCUSSION THREAD

**Discussion by Flmob** [thebaselab/codeapp#961] [General]
Just want to double check. I really like the app, thank you for your hard work! But it doesn't look completed yet and I do not see that there are some visible activity here.

What is the roadmap? Are there some new features in development? When they are planned to be released?

**Comment by bummoblizard** [comment]
Yes. The last update was 3 weeks ago. In fact, major improvements in SSH / Git were introduced in the Summer.

A list of TODOs can be seen at our Projects page: https://github.com/thebaselab/codeapp/projects?query=is%3Aopen

What feature would you like to see?

**Reply by Flmob** [reply]
I see now. Thank you.
For now I can leave somehow without 'npm i' not working properly.
But I'd really like to see support for next things:

 - Autocomplete
 - Typescript
 - Prettier
 - ESlint

Thank you for your feedback

**Reply by bummoblizard** [reply]
Many of the tools you mentioned can be directly installed using npm.
- TypeScript: `npm install -g typescript`
- Prettier: `npm install -g prettier`
- ESlint: `npm install -g eslint`

You can then run their CLI commands directly in the terminal, or through npm commands.

**Reply by Flmob** [reply]
It really should be mentioned in the docs -_-
But thank you. Will try
